Framework/Core/include/Framework/DeviceState.h

```diff
@@ -89,6 +90,8 @@ struct DeviceState {
   uv_loop_t* loop = nullptr;
   // The list of active timers which notify this device.
   std::vector<uv_timer_t*> activeTimers;
+  // The list of timers fired in this loop
+  std::set<uv_timer_t*> firedTimers;
```

Comment:
no sets in headers, please. Use an std vector

Reply:
sure, done